### ISSUE TITLE
How to ignore 'estimate' field?

### ISSUE THREAD

**Issue by yongli-abc** [duckdm/neowarrior.nvim#2]
Hi, I've already defined my own 'estimate' field in taskwarrior, now when I open neowarrior, I got:

```
Error executing Lua callback: .../packer/start/neowarrior.nvim/lua/neowarrior/Project.lua:63: attempt to perform arithmetic on field 'estimate' (a string value)
stack traceback:
        .../packer/start/neowarrior.nvim/lua/neowarrior/Project.lua:63: in function 'refresh'
        ...art/neowarrior.nvim/lua/neowarrior/ProjectCollection.lua:74: in function 'refresh'
        ...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:546: in function 'refresh'
        ...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:1695: in function 'open'
        ...r/start/neowarrior.nvim/lua/neowarrior/user_commands.lua:10: in function 'callback'
        ...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:916: in function <...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:915>
```

Can I get neowarrior to ignore this field?

**Comment by duckdm** [comment]
I've used the estimate field as a numeric field in taskwarrior my self but realise with your issue this does not always have to be the case so I've written a small fix to support the estimate field as a string instead. Obviously using estimate as string it will not be included in calculations for groups/projects, but should at least be presented on the "task page".

See https://github.com/duckdm/neowarrior.nvim/commit/8ce6dba6a18648b652fc137eb646abf051f5d72b

The fix should be available in development and release-0.4 branches, and will be included in 0.4.3. You can use a specific branch until it is included in release. Here's an example using lazy:
```lua
{
  "duckdm/neowarrior.nvim",
  branch = "develop",
  ...
}
```

**Comment by yongli-abc** [comment]
Thanks for the quick fix!
I've re-installed with your `develop` branch, but I got a different error now:

```
Error executing Lua callback: ...packer/start/neowarrior.nvim/lua/trambampolin/Buffer.lua:138: attempt to call field 'unpack' (a nil value)
stack traceback:
        ...packer/start/neowarrior.nvim/lua/trambampolin/Buffer.lua:138: in function 'print'
        ...k/packer/start/neowarrior.nvim/lua/trambampolin/init.lua:44: in function 'print'
        ...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:1778: in function 'list'
        ...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:1697: in function 'open'
        ...r/start/neowarrior.nvim/lua/neowarrior/user_commands.lua:10: in function 'callback'
        ...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:916: in function <...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:915>
```

**Comment by duckdm** [comment]
> Thanks for the quick fix! I've re-installed with your `develop` branch, but I got a different error now:
> 
> ```
> Error executing Lua callback: ...packer/start/neowarrior.nvim/lua/trambampolin/Buffer.lua:138: attempt to call field 'unpack' (a nil value)
> stack traceback:
>         ...packer/start/neowarrior.nvim/lua/trambampolin/Buffer.lua:138: in function 'print'
>         ...k/packer/start/neowarrior.nvim/lua/trambampolin/init.lua:44: in function 'print'
>         ...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:1778: in function 'list'
>         ...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:1697: in function 'open'
>         ...r/start/neowarrior.nvim/lua/neowarrior/user_commands.lua:10: in function 'callback'
>         ...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:916: in function <...cker/start/neowarrior.nvim/lua/neowarrior/NeoWarrior.lua:915>
> ```

I don't really understand why this is happening. What does your neowarrior config look like?

**Comment by yongli-abc** [comment]
```
    use {
        'duckdm/neowarrior.nvim',
        branch = 'develop',
        requires = {'nvim-telescope/telescope.nvim'},
        config = function()
            local nw = require('neowarrior')
            local home = vim.env.HOME
            nw.setup({
                report = "next",
                filter = "\\(due.before:2d or due: \\)",
                dir_setup = {
                    {
                        dir = home .. "/dev/nvim/neowarrior.nvim",
                        filter = "project:neowarrior",
                        mode = "tree",
                        expanded = true,
                    },
                }
            })
            vim.keymap.set("n", "<leader>nl", function() nw.open_left() end, { desc = "Open nwarrior on the left side" })
            vim.keymap.set("n", "<leader>nc", function() nw.open_current() end, { desc = "Open nwarrior below current buffer" })
            vim.keymap.set("n", "<leader>nb", function() nw.open_below() end, { desc = "Open nwarrior below current buffer" })
            vim.keymap.set("n", "<leader>na", function() nw.open_above() end, { desc = "Open nwarrior above current buffer" })
            vim.keymap.set("n", "<leader>nr", function() nw.open_right() end, { desc = "Open nwarrior on the right side" })
            vim.keymap.set("n", "<leader>nt", function() nw.focus() end, { desc = "Focus nwarrior" })
        end
    }

```

**Comment by yongli-abc** [comment]
I figured out. [table.unpack](https://github.com/duckdm/neowarrior.nvim/blob/bcce8526d8de13850e36014dfc6323766e9d9a1d/lua/trambampolin/Buffer.lua#L138) is only available after Lua 5.1.

I'm currently on Lua 5.1, which only has a global `unpack`. Will send a PR for fix.
